code/modules/food&drinks/kitchen machinery/gibber.dm

```diff
@@ -204,7 +204,7 @@
 	for (var/i=1 to meat_produced)
 		var/obj/item/weapon/reagent_containers/food/snacks/meat/slab/newmeat = new typeofmeat
 		var/obj/item/stack/sheet/animalhide/newskin = new typeofskin
-		newmeat.name = sourcename + newmeat.name
+		newmeat.name = "[sourcename] meat"
```

Comment:
this is gonna mess up any slab that isn't directly called 'meat' (skeleton and zombie slabs to be specific)